man/virt-xml.rst

```diff
@@ -152,6 +152,16 @@ These options decide what action to take after altering the XML. In the common c
 
     By default this also implies ``--define``.
 
+``--update-force``
+    **EXPERIMENTAL:** Forces a device update using libvirt's ``VIR_DOMAIN_DEVICE_MODIFY_FORCE`` flag.
```

Comment:
usually there are no direct mentions in the virt-manager/virt-install docs of the underlying libvirt APIs used; since it is not "usable" for users, I'd not mention the flag used